modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/fragments/raylife/fragments/congrats-information/index.css

```diff
@@ -29,6 +29,14 @@
 	position: static;
 }
 
+.congrats-info-price-discount{
```

Comment:
Is there a father class for `.congrats-info-price-discount`. If there is, use it to encapsulate it.

Reply:
I realized that no element would be reused from the parent component, so I decided to work with a different class. I just changed the class name.

Reply:
Done!